app/messages/service_credential_binding_create_message.rb

```diff
@@ -0,0 +1,33 @@
+module VCAP::CloudController
+  class ServiceCredentialBindingCreateMessage < MetadataBaseMessage
```

Comment:
I think it might be a while before we implement metadata for bindings so having this here and documented when we will not be using might be confusing. I didn't find it mentioned in any tests so it should be easy to remove it for the moment and not mention it in the docs yet.

Reply:
Fair point, will remove it